src/components/Settings/Settings.js

```diff
@@ -30,18 +42,122 @@ export default class Settings extends React.Component {
             <option value="TestNet">TestNet</option>
             <option value="CozNet">CozNet</option>
             <option value="nOSLocal">nOS Local</option>
+
+            {
+              this.props.allNetworks.map((network) => {
+                return (
+                  <option
+                    key={network.neoscan}
+                    value={network.neoscan}
+                  >
+                    {network.name}
+                  </option>
+                );
+              })
+            }
           </select>
         </label>
+        <div className={styles.buttonContainer}>
+          Custom Neoscan Url: {this.props.currentNetwork}
+        </div>
+        {this.renderButtons()}
         {this.state.saved && <Saved /> }
       </div>
     );
   }
 
+  renderButtons = () => {
+    return (
+      <div>
+        <Button onClick={this.handleAddNewNetwork}>
+          {'Add custom network configuration'}
+        </Button>
+        <div className={styles.divider} />
+        <Button onClick={this.handleClearNetwork}>
+          {'Clear custom network configurations'}
+        </Button>
+      </div>
+    );
+  }
+
+  handleClearNetwork = () => {
+    this.props.clearNetworks();
+    this.props.setCurrentNetwork('TestNet');
+    this.props.alert('All custom network configurations cleared');
+  }
+
+  handleAddNewNetwork = () => {
+    this.props.confirm((
+      <div>
+        <Input
+          id="networkName"
+          type="text"
+          label="Network name"
+          placeholder="Network name"
+          onChange={this.handleChangeNetworkName}
+        />
+        <Input
+          id="networkURL"
+          type="text"
+          label="Network URL"
+          placeholder="Network URL"
+          onChange={this.handleChangeNetworkUrl}
+        />
+      </div>
+    ), {
+      title: 'New network configuration',
+      onConfirm: this.handleConfirmAddNetwork,
+      onCancel: noop
+    });
```

Comment:
🤔 Not something we have to address for this PR, but food for thought -- I'd like to come up with a way that we can encapsulate confirmation dialogs like this to prevent the parent component from rendering complex form details and managing modal state.

Reply:
This is exactly what I need in the other open PR, we should have this as a portal (or HOC?) for easy reusability in another PR :P

Reply:
Yup I agree, I feel bad having the parent component managing state for the modal, but struggle to come up with some way to have the modal manage its own state and be generic so that its reusable in the way like the confirm and alerts modal are. What be happen to discuss more.

Reply:
I'll add an issue to track this.  Thanks for the feedback, guys.

Reply:
Added #118.